src/main/scala/uk/gov/hmrc/http/HttpDelete.scala

```diff
@@ -30,9 +30,9 @@ trait HttpDelete
     with HttpHooks
     with Retries {
 
-  override def DELETE[O](url: String)(implicit rds: HttpReads[O], hc: HeaderCarrier, ec: ExecutionContext): Future[O] =
+  override def DELETE[O](url: String, headers: Seq[(String, String)])(implicit rds: HttpReads[O], hc: HeaderCarrier, ec: ExecutionContext): Future[O] =
```

Comment:
Would it be worth to keep the API backwards compatible?

Reply:
Do you mean like adding a default parameter?

Reply:
yes, that could be a good way to keep it backwards compatible